packages/theme-material-ui/README.md

```diff
@@ -0,0 +1,14 @@
+# Access theme data as Material UI Theme
+
+```js
+import lightGrayMediumDensityTheme from '@hig/theme-material-ui/build/mui/lightGrayMediumDensityTheme';
+
+console.log(lightGrayMediumDensityTheme);
```

Comment:
We should probably add more to the readme on how to use the themes for `material-ui`.

Reply:
I think as we get more components we should definitely do this.